doc/radosgw/s3select.rst

```diff
@@ -0,0 +1,268 @@
+===============
+ Ceph s3 select 
+===============
+
+.. contents::
+
+Overview
+--------
+
+    | The purpose of **s3 select** engine is to create an efficient pipe between user client to storage node (the engine should be close as possible to storage).
```

Comment:
purpose of the

Reply:
s/between...to storage node/between...and storage nodes/;

Reply:
Really what we should lead off with is that this is an implementation of s3-select, and provide a few remarks on the implementation.  In the remarks you could add that it has been designed as a module that can be executed in multiple Ceph contexts, including RGW and RGW CLS, and potentially other contexts in future.  And that the intent of the Ceph integration is to provide acceleration through pushdown execution (which is where your original paragraph starts).

Reply:
ok.
this specific PR stand for the integration of s3select sub-module and RGW  (and only per CSV objects)

indeed, the s3-select-sub-module could use as general purpose library , for different types of acceleration  (pushdown) , and maybe to provide some dynamic programing.

we can add those remarks in this doc , or maybe the sub-module doc.